File: helloworld/web.java
HttpServer server = vertx.createHttpServer();
int port = 8082;

server.requestHandler(request -> {
  // This handler gets called for each request
  HttpServerResponse response = request.response();
  response.putHeader("content-type", "text/plain");

  // Write to the response and end it
  response.end("Hello World!");
});

server.listen(port);

File: helloworld/web.js
var server = vertx.createHttpServer();
var port = 8080;

server.requestHandler(function (request) {
  // This handler gets called for each request
  var response = request.response();
  response.putHeader("content-type", "text/plain");

  // Write to the response and end it
  response.end("Hello World from Vert.x (and JavaScript)!");
});

server.listen(port);
